app/templates/buyers/brief_overview.html

```diff
@@ -84,10 +84,10 @@
               'description': {
                 'draft': '',
                 'live': 'Your requirements are open for applications until {}.'.format(brief.applicationsClosedAt | utcdatetimeformat),
-                'closed': 'Your requirements closed for applications on {}.'.format(brief.applicationsClosedAt | utcdatetimeformat),
-                'awarded': 'Your requirements closed for applications on {}.'.format(brief.applicationsClosedAt | utcdatetimeformat),
-                'cancelled': 'Your requirements closed for applications on {}.'.format(brief.applicationsClosedAt | utcdatetimeformat),
-                'unsuccessful': 'Your requirements closed for applications on {}.'.format(brief.applicationsClosedAt | utcdatetimeformat),
+                'closed': 'Done',
```

Comment:
Can you remove the overview link on line 47?

Reply:
Do you mean the `context` that goes above the h1? I've just double checked the design doc and it's there for the  'closed' status but not the others - I'll ask Laurence and see if this is intentional.

Reply:
The powers that be say get rid of it 😸 I'll add a commit now.